packages/account/src/Styles/account.scss

```diff
@@ -1132,6 +1227,14 @@ $MIN_HEIGHT_FLOATING: calc(
     }
 }
 
+.dc-modal__container_sent-email__modal {
+    @include mobile {
+        height: unset !important;
+        max-height: 100vh !important;
```

Comment:
Using `vh` in css on mobile without `Div100VhContainer` calculate offsets wouldn't take the bottom browser toolbars into calculation, so you'll end up with the browser toolbar covering the bottom portion of this modal.